css/base.scss

```diff
@@ -206,10 +206,34 @@ input[type=search] {
         }
       }
 
+      .methods {
+        @include display-flex();
+        @include flex-direction(row);
+        flex-wrap: wrap;
+        margin: 0 10px;
+
+        > p {
+          margin: 5px 0;
+        }
+      }
+
+      &.positive {
```

Comment:
I would recommend putting all these selectors on one line unless separate declarations are to be inserted at different levels.